lib/engines/bh.js

```diff
@@ -26,7 +38,7 @@ module.exports = function (opts) {
             return;
         }
 
-        filePath = U.fulfillName(name, opts.ext || this.ext, opts.source);
+        filePath = U.fulfillName(name, opts.ext || this.ext, opts.source, lang);
```

Comment:
Good idea. Look this https://github.com/remnev/express-bem/commit/746bc252e4bde6bd2374ecdd1f9cd6ce63d46bc5